tests/integration.rs

```diff
@@ -224,3 +225,22 @@ fn readlink_short(harness: Harness) {
 //
 //	assert_eq!(link, Path::new(&expected));
 //}
+
+#[rstest]
+fn sparse(harness: Harness) {
+	let d = &harness.d;
+
+	let file = File::open(d.path().join("sparse")).unwrap();
+	let st = file.metadata().unwrap();
+
+	assert_eq!(st.blocks(), 256);
+
+	// TODO: why is this different from `stat -f %z sparse` (536891392)?
+	assert_eq!(st.size(), 536903680);
```

Comment:
I get the correct value from `stat -f %z sparse` when I use the kernel's UFS driver.

Reply:
That's interesting, because I did `./script/mkimg.sh -p tmp` and then `stat -f '%z' tmp/sparse` and it gave me a different number. Not sure why